package.json

```diff
@@ -24,6 +25,9 @@
   "license": "ISC",
   "homepage": "https://github.com/gr3p1p3/transparent-proxy",
   "devDependencies": {
+    "express": "^4.18.2",
```

Comment:
How do you build your production-grade package? Do you include dev dependencies in it?